converttomarkdown.gapps

```diff
@@ -102,17 +102,31 @@ function processParagraph(index, element, inSrc, imageCounter, listCounters) {
   // Note that Markdown does not process within block-level HTML, so it probably 
   // doesn't make sense to add markup within tables.
   if (element.getType() === DocumentApp.ElementType.TABLE) {
-    textElements.push("<table>\n");
+    var nRows = element.getNumChildren();
     var nCols = element.getChild(0).getNumCells();
-    for (var i = 0; i < element.getNumChildren(); i++) {
-      textElements.push("  <tr>\n");
-      // process this row
+    if(nRows == 1 && nCols == 1) {
+      textElements.push("~~~\n");
+      textElements.push(element.getChild(0).getChild(0).getText() + "\n");
+      textElements.push("~~~\n");
+    } else {
+      for (var j = 0; j < nCols; j++) {
+        textElements.push("| " + element.getChild(0).getChild(j).getText());
+      }
+      textElements.push(" |\n");
       for (var j = 0; j < nCols; j++) {
-        textElements.push("    <td>" + element.getChild(i).getChild(j).getText() + "</td>\n");
+        textElements.push("| --- ");
+      }
+      textElements.push(" |\n");
+
+
+      for (var i = 1; i < nRows; i++) {
+        // process this row
+        for (var j = 0; j < nCols; j++) {
+          textElements.push("| " + element.getChild(i).getChild(j).getText());
```

Comment:
![image](https://user-images.githubusercontent.com/1792088/27078371-0f82e7f0-5066-11e7-8cee-db2f942fd4d2.png)

I think this is the reason why choosing <table> at first place.